.idea/externalDependencies.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ExternalDependencies">
+    <plugin id="Error-prone plugin" />
+    <plugin id="google-java-format" />
+  </component>
+</project>
```

Comment:
maybe newline here?

Reply:
IJ generated file when using the UI. gonna leave as is as I think it'll get reformatted any time it's changed from settings.

Reply:
sounds good